README.md

```diff
@@ -1,19 +1,66 @@
-<h1 align="center">ES Check ✔️</h1>
-<h5 align="center">Checks the JavaScript files version of ECMAScript (ES) with shell commands  🏆</h5>
+<p align="center">
+  <img src="https://yowainwright.imgix.net/gh/es-check.svg" alt="ES Check ✔️" width="120" />
```

Comment:
hmmmm – can you host the image somewhere else like camo?

Reply:
It's SVG, so no.

Reply:
I'm happy to do this asap, though!